runtime.md

```diff
@@ -9,8 +9,11 @@ $ funC [global-options] <COMMAND> [command-specific-options] <command-specific-a
 
 ## Global options
 
-None are required, but the runtime may support options that start with at least one hyphen.
-Global options may take positional arguments (e.g. `--log-level debug`), but the option parsing must be such that `funC <COMMAND>` is unambiguously an invocation of `<COMMAND>` for any `<COMMAND>` that does not start with a hyphen (including commands not specified in this document).
+None are required, but the runtime MAY support options that start with at least one hyphen.
+Global options MAY take positional arguments (e.g. `--log-level debug`).
+Command names MUST not start with hyphens.
```

Comment:
On Mon, Dec 07, 2015 at 04:52:47PM -0800, Mike Brown wrote:
> Your statements clarified for me how you read them, specifically
> with the future guarantees and the purpose of making easier to
> parse. I don't think we need to add any statements about future
> revisions or the rationale for the requirement. It's the pattern
> selected .. so it is what it is.

Works for me.  Even if we don't change the spec text, I think it's
worth being very explicit about the rationale and our intentions in
the commit message.  That way we don't have to rehash the reasoning if
we do decide that the spec needs clarification later.  Can you update
the commit message with something to cover the points made in [1,2]?

[1]: https://github.com/wking/oci-command-line-api/pull/8/files#r46898167
[2]: https://github.com/wking/oci-command-line-api/commit/527f3c629a7224144c1e5e41a8a197fe0eb5bc00#commitcomment-14835617